drafts/2019-11-06-1900011.adoc

```diff
@@ -157,6 +160,95 @@ MetadataBuilder mdb = Metadata.builder();
 mdb = mdb.withName("metricName").withType(MetricType.COUNTER)
   .reusable(resolveIsReusable());
 ----
+[#JMO]
+== Jaeger for MicroProfile OpenTracing
+
+OpenLiberty can use Jaeger as a tracing backend for MicroProfile OpenTracing. 
```

Comment:
Can you change all instances of "OpenLiberty" to "Open Liberty"? Never run together.